src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -18,450 +18,700 @@ should be included in the pipeline documentation.
+| `dti`       | Diffusion Tensor Imaging \[[Basser1994](#basser1994)\]                                                                                          | Single [parameter vectors](#data-param) image with 6 volumes in the order: *D<sub>xx</sub>*, *D<sub>xy</sub>*, *D<sub>xz</sub>*, *D<sub>yy</sub>*, *D<sub>yz</sub>*, *D<sub>zz</sub>*<br>OR<br>Tensor coefficients as [parameter vectors] image with parameter name "`tensor`";<br>Estimated *b*=0 intensity as [scalar](#data-scalar) image with parameter name "`bzero`"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |

```

Comment:
This one needs some rethinking. The NIFTI standard has a *different* way of encoding this kind of data, see: https://github.com/nipy/dipy/issues/1890. We might want to use that one, since it's the standards-compliant way. I don't think this generalizes to DKI, though.

Reply:
As discussed: we may not necessarily want to support the NIfTI standard format for encoding such data, since the NIfTI specification contains many other things that we likely more explicitly don't want to support. But it may depend on whether or not any softwares actually provide tensor data using this specification; if such are rare or non-existent, then I'd personally prefer to write this off under the 80/20 rule.

Reply:
Has this been addressed? If not, please open a new issue for it.

Reply:
My personal preference is to define tensor data in a way we devise that is consistent with the rest of BIDS Derivatives, rather than trying to propagate aspects of the NIfTI format that are not themselves consistently utilised. Conversion between the two should not be difficult in terms of volume permutation, but I don't know what proportion of softwares would actually permit setting / obeying NIfTI intent codes. I'm not against including support for such, but it would need to be exceptionally clear in the JSON that that specific format is utilised, rather than relying on the NIfTI intent codes alone.

This one's slightly complex so happy to defer to a dedicated issue.

Reply:
Dedicated issue is #17